vesc_driver/src/vesc_driver.cpp

```diff
@@ -58,6 +59,36 @@ VescDriver::VescDriver(ros::NodeHandle nh,
 
   // create a 50Hz timer, used for state machine & polling VESC telemetry
   timer_ = nh.createTimer(ros::Duration(1.0/50.0), &VescDriver::timerCallback, this);
+
+  // decide whether or not we want to publish rotor positions
+  // this is enabled with by sending a "set detect" packet with a mode
+  // "encoder" is probably the most useful
+  std::string rotor_position_source;
+  if (private_nh.getParam("rotor_position_source", rotor_position_source)) {
+    disp_pos_mode mode = DISP_POS_MODE_NONE;
```

Comment:
@mboulet 
one small thing i was wondering about: do you think this mode should be abstracted into vesc_interface, perhaps through a separate function for each display mode?
it feels slightly more "correct" to do so but would introduce some bloat